build.sh

```diff
@@ -2,30 +2,43 @@
 
 set -e
 
-cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+if [ -z "$1" ]; then
+  echo "Usage: build.sh destdir"
+  exit 1
+fi
+
+NVML_SRC_DIR=`dirname $(readlink -f $0)`
```

Comment:
Use `$()` over legacy backticks, quote `$0` and result of `readlink` to prevent word splitting.
```suggestion
NVML_SRC_DIR=$(dirname "$(readlink -f "$0")")
```